.bitmap

```diff
@@ -631,6 +631,12 @@
         "mainFile": "index.ts",
         "rootDir": "scopes/ui-foundation/uis/corner"
     },
+    "ui/debbie": {
```

Comment:
@debs-obrien , don't forget to remove this :)

Reply:
ooops thanks